sunpy/database/tests/test_attrs.py

```diff
@@ -443,5 +447,5 @@ def test_walker_create_time(vso_session):
             fileid=u'/hessidata/2011/09/19/hsi_20110919_233340',
             observation_time_start=datetime(2011, 9, 19, 23, 33, 40),
             observation_time_end=datetime(2011, 9, 20, 1, 9, 20),
-            instrument=u'RHESSI', size=-1.0, wavemin=0.4132806579880238,
```

Comment:
Stupid question, what happened to size?

Reply:
iirc, it's an optional parameter which is no longer always in the response. I.e. it's only there if it is set.